app/models/data_update_script.rb

```diff
@@ -29,7 +29,7 @@ def scripts_to_run?
 
     def filenames
       Dir.glob("*.rb", base: DIRECTORY).map do |f|
-        Pathname.new(f).basename(".rb").to_s
+        File.basename(f, ".rb")
```

Comment:
According to my testing, this looks to return the same thing but is a bit simpler to read.